go/vt/vttablet/tabletserver/query_engine_test.go

```diff
@@ -98,7 +98,7 @@ func TestGetPlanPanicDuetoEmptyQuery(t *testing.T) {
 
 	ctx := context.Background()
 	logStats := tabletenv.NewLogStats(ctx, "GetPlanStats")
-	_, err := qe.GetPlan(ctx, logStats, "", false)
+	_, err := qe.GetPlan(ctx, logStats, "", false, false /* inReservedConn */)
```

Comment:
does it make sense to add tests here with true?

Reply:
query_engine does not work with reserved connections.